dist/styles/base.scss

```diff
@@ -156,6 +156,7 @@
 
 .fr_field_wrapper {
   position: relative;
+  clear: both;
```

Comment:
This fix is to prevent several field types from floating over their labels while within a nested group. This clears the float so they appear on the next line instead, like is normal for the rest of the form.

Example:
<img width="492" alt="screen shot 2017-12-14 at 2 33 23 pm" src="https://user-images.githubusercontent.com/538235/34013096-c9a30cf4-e0db-11e7-85ee-f476e77344d9.png">
<img width="579" alt="screen shot 2017-12-14 at 2 34 09 pm" src="https://user-images.githubusercontent.com/538235/34013120-e25148d8-e0db-11e7-8285-dedea79a4069.png">